src/App.js

```diff
@@ -0,0 +1,29 @@
+import './App.css';
+import Home from './Container/Home/Home';
+import About from './Container/About/About';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
+
+function App() {
+  return (
+    <>
+        <Router>
+        <div>
+          <h2>Analectica - Frontend</h2>
+          <nav>
+          <ul >
+            <li><Link to={'/'}> Home </Link></li>
```

Comment:
```suggestion
            <li><Link to="/"> Home </Link></li>
```

No need to use `{}` as it's a simple string.